cmd/show-coverage.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+  "fmt"
+  "strings"
+  "io/ioutil"
+  "encoding/json"
+
+  "github.com/pkg/errors"
+  "github.com/spf13/cobra"
+)
+
+var showCoverageCmd = &cobra.Command{
+  Use:   "show-coverage",
+  Short: "Show coverage results in standard output",
+  RunE: func(cmd *cobra.Command, args []string) error {
+    if len(args) == 0 {
+      return errors.New("you must pass in one file with the coverage results")
+    }
+
+    dat, err := ioutil.ReadFile(args[0])
+    if err != nil {
+      return errors.New("could not open input file")
+    }
+
+    var result map[string]interface{}
+    json.Unmarshal([]byte(string(dat)), &result)
+    line_counts := result["line_counts"].(map[string]interface{})
+    header := "Coverage: %.2f%% (%d/%d lines covered, %d missing)"
+    fmt.Println(fmt.Sprintf(header, result["covered_percent"], int(line_counts["covered"].(float64)), int(line_counts["total"].(float64)), int(line_counts["missed"].(float64))))
```

Comment:
The body of this function is a bit dense. Do you mind extracting a few new functions to smooth out the read?

Reply:
Done